js/api.js

```diff
@@ -0,0 +1,66 @@
+const base_url = 'https://js-todo-list-9ca3a.df.r.appspot.com';
+
+const option = {
+    post: (contents) => ({
+        method: 'POST',
+        headers: { 'Content-Type': 'application/json' },
+        body: JSON.stringify(contents),
+    }),
+    
+    put: (contents) => ({
+        method: 'PUT',
+        headers: { 'Content-Type': 'application/json' },
+        body: JSON.stringify(contents),
+    }),
+
+    delete: () => ({
+        method: 'DELETE',
+    }),
+
+};
+
+const request = async (url, option = {}) => {
+    try{
+        const response = await fetch(`${base_url}${url}`, option);
+        if(!response.ok){
+            throw new Error(response.status);
+        }
+        return await response.json();
```

Comment:
```suggestion
        return response.json();
```
await 은 불필요한 키워드 일 것 같습니다. 
request 를 사용하는 입장에서 async가 붙은 함수는 항상 promise 를 반환해 주기 때문에 await 으로 해체해 주어도 다시 promise 로 감싸지기 때문입니다. 
이외에 await 으로 사용한 다른 의도가 있을까요?